### DISCUSSION TITLE
Can't find JavaScript under Environment

### DISCUSSION THREAD

**Discussion by solidityb** [smartcontractkit/full-blockchain-solidity-course-py#1712] [Q&A] [answered]
Can someone help me here?

![image](https://user-images.githubusercontent.com/114201152/191878689-b9b1ecdc-6283-480a-9c43-3e375231af2c.png)

**Comment by solidityb** [comment]
![image](https://user-images.githubusercontent.com/114201152/191878770-0c2c1624-7c81-4736-9b82-8e7ca2873bfc.png)

**Reply by Niferu** [reply]
Hello @solidityb 
For this one you have just named "SimpleStorage.Sol" instead of "SimpleStorage.sol" it does matter and that is why your formatting dosent work here.

**Reply by solidityb** [reply]
Thanks!

**Comment by solidityb** [comment]
![image](https://user-images.githubusercontent.com/114201152/191889897-87d097a0-7aac-46cc-b5d9-78dbb405c2c4.png)
also need help with this.

1. my // doesn't show italic sentences after this
2. i couldn't retrieve the number i type in store

**Reply by Niferu** [reply]
Hello @solidityb 

Have you clicked on "store" button to actually perform this action? Only after clicking function button it's code will be executed.

**Reply by solidityb** [reply]
Thanks so much. You've been a great help

**Comment by solidityb** [comment]
![image](https://user-images.githubusercontent.com/114201152/191895020-4f4a98ea-3ea8-4bae-a23e-67a02e9656e5.png)
why is there a red mark. what error did i make

**Reply by Niferu** [reply]
Hello @solidityb 

You have a typo in addperson function as you should push "favoriteNumber" instead of "_favoriteNumber".

**Reply by cromewar** [reply]
Nice catch @Neftyr

**Reply by solidityb** [reply]
function addperson(string memory _name, uint256 favoriteNumber) public {
        people.push(People({favoriteNumber: _favoriteNumber, name: _name}));
    }

how should I change it?

**Reply by Niferu** [reply]
@solidityb 
You should change it into:
```solidity
function addperson(string memory _name, uint256 favoriteNumber) public {
people.push(People({favoriteNumber: favoriteNumber, name: _name}));
}
```

**Comment by Niferu** [comment, marked as answer]
Hello @solidityb 

You should just use "Remix VM (London)" and it will work perfectly fine.

**Reply by solidityb** [reply]
Thanks. Thought so. Was also confused with video having suggestions and I don't have, and we had different colors. I figure I will sort that out later. You have been so helpful. Appreciate it. I have zero coding experience and have just practiced on some python coding.

**Reply by Niferu** [reply]
Glad I could help :)
